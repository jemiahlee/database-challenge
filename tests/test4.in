set var this_is_a_string
begin
get var
numequalto this_is_a_string
numequalto 78
set var 78
numequalto this_is_a_string
numequalto 78
get var
unset var
get var
numequalto 78
rollback
get var
numequalto 78
numequalto this_is_a_string
END
