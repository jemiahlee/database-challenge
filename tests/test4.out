this_is_a_string
1
0
0
1
78
NULL
0
this_is_a_string
0
1
